### DISCUSSION TITLE
请问会出一个删除某些数据的功能吗

### DISCUSSION THREAD

**Discussion by qiujubin** [shion-app/shion#58] [Q&A]
<img width="245" alt="屏幕截图 2024-10-18 113243" src="https://github.com/user-attachments/assets/5469f961-2fdc-483f-ae2a-e3103e99ae47">

比如这个异常的数据，可不可以采用，比如右键删除之类的方法，来将它去掉

**Comment by hanaTsuk1** [comment]
在回答这个之前，我要获取更详细的信息
你这个数据怎么显示4个小时？怎么弄出来的？

自动监听的数据我没有添加删除的选项，因为一般不需要做操作，数据有问题就是bug需要修复

**Reply by qiujubin** [reply]
不清楚怎么弄出来的，还是第一次见到这样的。
<img width="536" alt="1" src="https://github.com/user-attachments/assets/ce295b71-5a75-42af-bac3-2154b466e7c2">

但是数据库里的是这样的：
<img width="624" alt="3" src="https://github.com/user-attachments/assets/7a80c518-abf6-4597-a2b3-acf8073840b5">



但是软件有点对不上

**Reply by qiujubin** [reply]
啊不好意思，sql语句写错了，应该是p.id才对，那样的话软件对应没错

**Reply by hanaTsuk1** [reply]
了解，我之后看看相关代码，我把这个转到issue
如果再次遇到这个情况，麻烦上传一下日志文件（要是退出了，重新打开软件，之前的日志不会保留）

既然你会数据库，那先用一下临时的解决办法，把activity的数据deleted_at改为非0就算删除了，你也可以改成当前时间戳

**Reply by hanaTsuk1** [reply]
https://github.com/shion-app/shion/issues/59

**Reply by qiujubin** [reply]
好的

**Reply by hanaTsuk1** [reply]
感谢配合🙏
